environment/integration/helpers.go

```diff
@@ -169,7 +203,7 @@ func initializeDaggerOnce(t *testing.T) {
 	})
 
 	if daggerErr != nil {
-		t.Skipf("Skipping test - Dagger not available: %v", daggerErr)
+		t.Fatalf("Dagger not available: %v", daggerErr)
```

Comment:
🐛 **Bug (High):** Changed behavior from skipping tests to failing them when Dagger/Docker is unavailable. This will break CI environments without Docker. Should maintain original skip behavior unless intentionally changing test requirements.

```suggestion
		t.Skipf("Skipping test - Dagger not available: %v", daggerErr)
```

Reply:
It's false. This is intended